vue.config.js

```diff
@@ -8,5 +8,6 @@ module.exports = {
     apollo: {
       lintGQL: false
     }
-  }
+  },
+  publicPath: "./"
```

Comment:
Required for the script and style tags to use relative paths, which is necessary for them to load at all in GitHub Pages (at least until we have a solid subdomain - but even then relative will work)